templates/terraform/k8s-master/main.tf

```diff
@@ -30,7 +22,10 @@ resource google_container_cluster master {
   # CIS compliance: stackdriver monitoring
   monitoring_service = "monitoring.googleapis.com/kubernetes"
 
-  min_master_version = data.google_container_engine_versions.cluster_versions.latest_master_version
```

Comment:
This was a constant source of noise in our `apply` outputs, because GKE auto-upgrades its master nodes and there's no way to disable it.

Specifying a release channel is a similar way to achieve the spirit of what we were trying to do here. The `REGULAR` channel still auto-upgrades a few times per month, but it's not as aggressive as `RAPID` or having no release channel. For example, our clusters had auto-upgrades to k8s `1.15`, but the `REGULAR` channel is still working through `1.14`